src/js/components/SingleImage.vue

```diff
@@ -240,6 +240,15 @@ export default {
       this.drawingMode = this.drawingMode === 'draw' ? 'erase' : 'draw'
     }
   },
+  mounted() {
+    if (this.$route.params.imageID) {
```

Comment:
I just checked through this, since your suggestion seemed to have found a bug.  However, the two conditions check for different things and do different things: the first condition checks for an imageID and loads up the relevant images if there is one, the second check for an artID and loads up the artefact path if there is one.  They can both be true, in which case they both have their own separate functions that fire the ajax requests and deal with the return data.  It is only a bit unfortunate when we have an artID but no imageID, in which case you will see the polygon of an artefact, but no image underneath.  I added a TODO to think about avoiding that case (the menu system doesn't allow it, but someone could always manually type it in).